app/services/load_testing_urls_writer.rb

```diff
@@ -0,0 +1,43 @@
+class LoadTestingUrlsWriter
+  LOAD_URLS_FILE_NAME = "load_urls.txt"
+
+  def initialize(env:, num_different_books: 3, urls_per_book: 3)
+    @num_different_books = num_different_books
+    @urls_per_book = urls_per_book
+    @env = env
+  end
+
+  def call
+    book_ids_to_use = sample_some_books_ids
+    File.open(LOAD_URLS_FILE_NAME, 'w') do |file|
+      book_ids_to_use.each do |book_id|
+        terms_to_use = find_terms_in_book_to_search_for(book_id)
+        write_urls(file, book_id, terms_to_use)
```

Comment:
Can we configure siege to take URLs in random order? if not this will write all URLs for one book, then the next, and so on, which won't be as realistic of a test.

Reply:
yes, siege has the -i flag that uses the URLs in random order